Jenkinsfile

```diff
@@ -199,3 +73,11 @@ pipeline {
     }
   }
 }
+
```

Comment:
this is not needed. It's using the one from the shared lib.

Reply:
i replaced the JenkinsFile with the one on dev with the only differences now being  a change to jdk version and a separate stage for the minicluster module